src/settings.js

```diff
@@ -1742,26 +1740,29 @@ var MIN_FIREFOX_VERSION = 65;
 // NOTE: Emscripten is unable to produce code that would work in iOS 9.3.5 and
 // older, i.e. iPhone 4s, iPad 2, iPad 3, iPad Mini 1, Pod Touch 5 and older,
 // see https://github.com/emscripten-core/emscripten/pull/7191.
+// MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
 var MIN_SAFARI_VERSION = 120000;
 
 // Specifies the oldest version of Internet Explorer to target. E.g. pass -s
 // MIN_IE_VERSION = 11 to drop support for IE 10 and older.
 // Internet Explorer is at end of life and does not support WebAssembly.
-// MAX_INT (0x7FFFFFFF) specifies that target is not supported.
+// MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
 var MIN_IE_VERSION = 0x7FFFFFFF;
 
 // Specifies the oldest version of Edge (EdgeHTML, the non-Chromium based
 // flavor) to target. E.g. pass -sMIN_EDGE_VERSION=40 to drop support for
 // EdgeHTML 39 and older.
 // Edge 44.17763 was released on November 13, 2018
+// MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
 var MIN_EDGE_VERSION = 44;
 
 // Specifies the oldest version of Chrome. E.g. pass -sMIN_CHROME_VERSION=58 to
 // drop support for Chrome 57 and older.
 // Chrome 75.0.3770 was released on 2019-06-04
+// MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
```

Comment:
Does -1 really work?  If so maybe just stick to that and drop the 0x7FFFFFFF magic?

Reply:
Yes, it does work. We have code to convert them in emcc.py.

I agree it makes sense to simplify this, and -1 is probably better. I can look into that as a followup.

Reply:
Or I guess this is so small I could wait and do it here.